### PR TITLE
Livesite settings page

### PR DESCRIPTION
**Pull request by ianmah**
image
image

### CODE REVIEW

File: pages/Livesite/settings.js [hunk 1]
```diff
@@ -1,23 +1,152 @@
...
+            {`Last edited by ${livesiteData.lastEditedBy} at ${formatDate(
+              livesiteData.lastEdited?.seconds
+            )}`}
+          </p>
+          <CardButtonContainer>
+            <Button type={EDIT} onClick={() => setisEditing(true)} />
+          </CardButtonContainer>
+        </CardHeader>
+        <CardContent>
+          {isEditing ? (
```
Comment (jackyzha0) [inline pages/Livesite/settings.js:98]: still confused as to what the `isEditing` flag here does

File: utility/firebase.js [hunk 2]
```diff
@@ -268,6 +269,18 @@ export const uploadSponsorImageToStorage = async (website, file) => {
...
   }
 };
+
+export const uploadLivesiteLogoToStorage = async (file) => {
+  try {
+    const ref = storage.ref(`logo.svg`);
+    const uploadData = await ref.put(file);
+    return uploadData.ref.getDownloadURL();
+  } catch (e) {
+    alert(e);
```
Comment (jackyzha0) [inline utility/firebase.js:279]: linter angry

**Review by jackyzha0 — COMMENTED** [review]
do we also want to be able to edit Hacking Period now that we're moving that to the Livesite object instead of DayOf events?

File: pages/Livesite/settings.js (hunk 1 above)
Comment (ianmah) [reply pages/Livesite/settings.js:98]: In the screenshot u can see the editing state and the non editing state

File: utility/firebase.js (hunk 2 above)
Comment (ianmah) [reply utility/firebase.js:281]: 😞

**Review by andykwan1124 — APPROVED** [review]

**Review by jackyzha0 — APPROVED** [review]
🕹️ mmm

**Pull request merged**